docs/how-to/uc-image-creation.rst

```diff
@@ -0,0 +1,307 @@
+How to create a real-time Ubuntu Core image
+===========================================
+
+.. note::
```

Comment:
Hey @farshidtz , I see this might be a snippet that would be reused quite a bit.

We can add a file under `<repo>/docs/reuse`, e.g. `snippets.txt`.

And then add snippets like this (in RST or MD syntax) and reference it using the `include` directive.
https://github.com/canonical/sphinx-docs-starter-pack/blob/main/doc-cheat-sheet.rst?plain=1#L207

Reply:
I plan to do some refactoring (merging the uc and classic config instructions) and then this exact note will no longer be repeated.